float64 tot_for_x
float64 tot_for_y
float64 planner_x
float64 planner_y
